### ISSUE TITLE
Getting environment variable in arguments?

### ISSUE THREAD

**Issue by jpetitcolas** [kimmobrunfeldt/concurrently#52]
How can we handle environment variables from a `concurrently` command?

I need the full path of working directory in order to launch a Docker container. So, I need to use the `$PWD` environment variable. 

With `concurrently`:

``` js
{
    "scripts": {
        "start": "concurrently 'echo ${PWD}'"
    }
}
```

It outputs `${PWD}`. Yet, if I don't use concurrently:

``` js
{
    "scripts": {
        "start": "echo ${PWD}"
    }
}
```

It works. Any idea on how I can solve this? :)

**Comment by kimmobrunfeldt** [comment]
Good question, I thought environment variables would be available in the commands. This should be fixed.

**Comment by kimmobrunfeldt** [comment]
Could you test if this has been fixed by installing 3.0.0-dev version: `npm i -g concurrently@3.0.0-dev` ? 
It should be better now because it is using https://github.com/kimmobrunfeldt/spawn-default-shell for spawning shell. At least simple echo worked for me.

**Comment by kimmobrunfeldt** [comment]
These issues should be now fixed in 3.0.0-rc1. You can test by installing it: `npm i -g concurrently@3.0.0-rc1`. Please open a new issue to concurrently repo if this is not the case. I'll release the 3.0.0 soonish.
